

2023.05.31 10:05:06 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:06:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:07:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:08:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:09:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:10:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:12:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:14:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:15:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:17:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:19:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:23:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:25:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:26:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:28:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:30:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:35:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:36:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:38:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:41:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:42:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:43:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:44:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:45:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:46:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:47:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:49:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:51:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:52:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:53:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:54:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:55:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:56:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:57:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:58:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 10:59:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:00:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:01:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:02:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:04:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:05:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:07:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:08:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:09:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:10:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:11:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:12:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:13:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:14:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:15:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:17:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:19:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:21:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:22:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:23:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:25:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:28:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:29:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:30:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:31:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:33:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:34:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:35:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:36:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:38:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:40:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:41:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:42:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:43:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:44:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:46:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:47:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:49:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:51:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:52:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:54:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:55:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:56:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:57:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 11:59:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:01:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:04:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:05:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:07:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:08:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:09:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:10:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:12:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:13:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:14:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:15:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:16:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:18:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:19:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:20:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:21:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:25:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:28:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:29:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:31:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:42:26 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:43:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:44:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 12:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
2023.05.31 13:14:46 : Error : [Errno 60] Operation timed out

Traceback (most recent call last):
  File "emailProcess.py", line 394, in emailProcess
    unread_mails = imbox.messages(unread=True)
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 100, in messages
    return messages_class(connection=self.connection,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/messages.py", line 37, in __init__
    self._uid_list = self._query_uids(**kwargs)
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/messages.py", line 48, in _query_uids
    _, data = self.connection.uid('search', None, query_)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 881, in uid
    typ, dat = self._simple_command(name, command, *args)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1205, in _simple_command
    return self._command_complete(name, self._command(name, *args))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1022, in _command_complete
    typ, data = self._get_tagged_response(tag, expect_bye=logout)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1148, in _get_tagged_response
    self._get_response()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1050, in _get_response
    resp = self._get_line()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1158, in _get_line
    line = self.readline()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 314, in readline
    line = self.file.readline(_MAXLINE + 1)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
TimeoutError: [Errno 60] Operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "emailProcess.py", line 423, in emailProcess
    server.send_mail(sent_from[0]['email'], mail)
UnboundLocalError: local variable 'sent_from' referenced before assignment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 981, in _command
    self.send(data + CRLF)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 323, in send
    self.sock.sendall(data)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 1204, in sendall
    v = self.send(byte_view[count:])
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 1173, in send
    return self._sslobj.write(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 425, in emailProcess
    continue
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 49, in __exit__
    self.logout()
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 52, in logout
    self.connection.close()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 466, in close
    typ, dat = self._simple_command('CLOSE')
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1205, in _simple_command
    return self._command_complete(name, self._command(name, *args))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 983, in _command
    raise self.abort('socket error: %s' % val)
imaplib.abort: socket error: [Errno 32] Broken pipe


2023.05.31 13:16:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:19:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:20:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:25:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:26:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:27:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:28:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:29:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:30:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:32:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:36:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:38:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:40:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 13:41:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:42:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:43:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:44:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:47:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:49:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:51:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:52:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:54:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:55:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:56:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:57:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 13:59:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:01:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:03:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:04:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1300, in _create_socket
    sock = IMAP4._create_socket(self)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 293, in _create_socket
    return socket.create_connection((host, self.port))
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known


2023.05.31 14:07:04 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:08:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:10:46 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:12:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:14:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:15:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:18:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:19:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:22:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:23:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:25:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:26:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:27:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:28:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:31:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:33:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:36:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:38:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:41:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:42:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:43:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:44:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:47:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:49:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:51:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:52:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:54:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:55:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:56:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:57:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 14:59:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:01:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:04:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:07:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:08:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:09:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:10:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:12:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:14:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:15:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:18:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:19:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:22:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:23:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:24:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:25:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:26:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:28:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:29:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:31:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:35:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:36:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:37:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:38:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:39:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:40:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:41:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:42:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:43:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:44:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:45:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:46:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:47:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:49:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:50:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:51:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:52:04 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:53:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:54:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:55:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:56:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:57:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 15:59:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:01:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:04:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:07:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:08:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:09:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:10:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:11:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:12:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:13:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:14:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:15:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:16:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:17:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:19:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:25:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:27:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:28:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:32:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:36:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:38:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:41:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:42:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:43:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:44:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:47:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:49:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:50:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:51:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:52:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:54:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:55:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:56:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:57:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 16:59:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:01:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:04:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:06:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:07:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:08:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:09:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:10:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:12:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:14:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:15:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:19:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:25:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:28:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:32:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:36:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:37:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:38:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:41:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:42:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:43:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:44:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:47:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:48:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:49:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:51:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:52:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:54:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:55:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:56:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:57:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 17:59:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:01:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:04:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:07:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:08:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:09:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:10:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:12:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:14:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:15:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:19:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:25:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:28:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:47:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:49:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:51:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:52:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:54:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:55:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:56:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:57:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:58:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 18:59:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:01:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:02:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:04:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:07:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:08:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:09:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:10:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:11:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:12:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:14:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:15:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:17:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:18:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:19:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:22:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:23:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:25:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:28:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:36:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:38:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:41:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:42:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:43:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:44:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:47:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:49:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:50:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:51:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:52:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:53:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:54:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:55:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:56:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:57:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:58:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 19:59:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:00:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:01:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:02:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:04:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:05:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:06:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:07:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:08:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:09:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:10:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:11:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:12:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:13:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:14:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:15:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:17:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:18:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:19:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:22:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:25:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:27:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:28:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:30:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:32:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:34:20 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:36:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:38:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:41:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:42:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 20:43:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
Traceback (most recent call last):
  File "emailProcess.py", line 429, in <module>
    emailProcess()
  File "emailProcess.py", line 392, in emailProcess
    with Imbox('imap.qq.com', sender_qq, pwd, ssl=True) as imbox:
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imbox.py", line 21, in __init__
    self.server = ImapTransport(hostname, ssl=ssl, port=port,
  File "/Users/moshaocong/Library/Python/3.8/lib/python/site-packages/imbox/imap.py", line 18, in __init__
    self.server = IMAP4_SSL(self.hostname, self.port, ssl_context=ssl_context)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1297, in __init__
    IMAP4.__init__(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 198, in __init__
    self.open(host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1310, in open
    IMAP4.open(self, host, port)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 303, in open
    self.sock = self._create_socket()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/imaplib.py", line 1301, in _create_socket
    return self.ssl_context.wrap_socket(sock,
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
TimeoutError: [Errno 60] Operation timed out


2023.05.31 21:12:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:13:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:14:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:15:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:16:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:17:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:18:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:19:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:20:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:21:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:22:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:23:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:24:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:25:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:26:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:27:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:28:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:29:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:30:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:31:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:32:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:33:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:34:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:35:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:36:00 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:37:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:38:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:39:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:40:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:41:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:42:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:43:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:44:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:45:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:46:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:47:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:48:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 21:49:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 22:37:41 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 22:39:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:03:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:04:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:37:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:38:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:39:01 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:40:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:41:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:42:03 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:43:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----


2023.05.31 23:44:02 : 邮件自动回复已启动
login Success
-----开始接收并处理邮件-----
